docs/devs/Use-JerryScript-Debugger.md

```diff
@@ -23,12 +23,26 @@ The file argument is ignored in this case, therefore doesn't need to be specifie
 thus, there's no need to restart the environment if the remote source is changed.
 **Important note**: Remote sources must be sent in correct order! IoT.js compiles them in the order they are received, so file(s) used with `require` should be sent first, and the file(s) using them after.
 
+#### Select channel and Protocol
+
+There are two available extension-provided channel, websocket and rawpacket, and two procotol, tcp and serial. Each initializes the debugger and blocks until a client connects. If you want to specify the debugger channel (default: websocket) or protocol (default: tcp) over the communication you can do with the `--debug-channel [websocket|rawpacket]` and `--debug-protocol [tcp|serial]` options:
```

Comment:
I found a typo: procotol -> protocol. Please also check the plural form for channel and protocol.